### DISCUSSION TITLE
Commercial help

### DISCUSSION THREAD

**Discussion by jainvikasblr** [fabricjs/fabric.js#9116] [Q&A]
How can I get commercial help if I need customization for specific needs in Fabricjs like supporting multi-touch to be enabled on Android supported HD Monitors?
Rendering performance all multi-touch points will be key aspect. Note that such monitors allow simultaneous 20 touch points as depicted in youtube video (show cases 10 touch points): https://youtu.be/nA89-16wJ54?t=220

Appreciate the  response from community if indicated that it is not possible or max how many touch points possible

**Comment by ShaMan123** [comment]
You can send me an email, shacharnen@gmail.com
